### ISSUE TITLE
Переопределение стандартного com_media для доп.полей

### ISSUE THREAD

**Issue by AlekVolsk** [Quantum-Manager/tracker#114]
**Describe the bug**
Не переопределяет стандартный медиа-менеджер для дополнительный полей (com_fields) типа img и imglist ()

**System Information (please complete the following information):**
 - Extensions version :2.0.0
 - PHP version: 8.1
 - Joomla: 4.1.x

**Comment by dmitriitux** [comment]
да, пока не сделал, отказался на эту версию, там нужно делать новые поля вообще, я выпущу фикс релиз, а потом буду закрывать что не мигрировал на j4

**Comment by sokoloff-rv** [comment]
Добрый день! В первую очередь скажу спасибо за отличный файловый менеджер ) А теперь задам вопрос по делу: не подскажете, когда примерно планируется выпустить новую версию, которая будет перехватывать вызов медиа-менеджера для полей с типом "media" в модулях Joomla 4? А то стандартный менеджер прям совсем не устраивает, чисто из-за этого момента всё ещё не мигрировал на четверку.
